app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
```

Comment:
It's not that hard :-) I can guide you.

And we wouldn't merge something just to remove it subsequently.